documentation/performance/attribute-memory-usage.html

```diff
@@ -60,7 +60,14 @@ <h3 id="example">Example</h3>
 with various data types and collection types.
 It also contains estimates for how many documents a 48GB RAM node can hold, taking initialization into account.
 </p>
-
+<p>
+When <em>resizing</em> attributes, both current and new attribute will temporarily reside in memory.
+Hence, increasing an attribute means more than doubling the memory used -
```

Comment:
I thought this was changed, at least for multivalue attributes? Ref Geir/Tor work last Q

Reply:
Yes, removed it. I